mysql-test/t/all_persisted_variables.test

```diff
@@ -39,7 +39,7 @@
 --source include/have_binlog_format_row.inc
 
 let $total_global_vars=`SELECT COUNT(*) FROM performance_schema.global_variables where variable_name NOT LIKE 'ndb_%'`;
-let $total_persistent_vars=359;
+let $total_persistent_vars=413;
```

Comment:
Do we have 413-359 encryption related variables?

Reply:
No, but we never updated this testcase for percona server - so counting all of our variables, that number looked realistic.